src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlCommandListExtended.cs

```diff
@@ -0,0 +1,80 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using static Interop.VoiceControlManager;
+using static Interop.ControlCommandExpand;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class represents the Extended Command List
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class VoiceControlCommandExtendedList : VoiceCommandList
```

Comment:
You must named a class to indicate its purpose.
`Extended` is not proper, You need describe what is extend from base class